app/Platform/Services/GameTopAchieversService.php

```diff
@@ -171,8 +172,13 @@ private function convertPlayerGames(Collection $playerGames): array
     {
         $retval = [];
         foreach ($playerGames as $playerGame) {
+            if (!$playerGame->relationLoaded('user') && $playerGame->user_id) {
+                $playerGame->load('user');
+            }
```

Comment:
Is this different than `$playerGame->loadMissing('user');`?

Reply:
It is, but we can simplify this to use `->loadMissing()`. Updated in latest.